roles/metrics-export.lua

```diff
@@ -125,6 +125,20 @@ local http_handlers = {
 -- It is used as an error string with the predefined order.
 local http_supported_formats_str = "json, prometheus"
 
+local function validate_endpoint_metrics(metrics)
+    if type(metrics) ~= 'table' then
+        error("http endpoint 'metrics' must be a table, got " .. type(metrics), 4)
+    end
+
+    if is_array(metrics) then
+        error("http endpoint 'metrics' must be a map, not an array", 3)
+    end
+
+    if metrics.enabled ~= nil and type(metrics.enabled) ~= 'boolean' then
+        error("http endpoint metrics 'enabled' must be a boolean, got " .. type(metrics.enabled), 4)
+    end
```

Comment:
Since it is called from a 4 level of nesting.
```suggestion
    if type(metrics) ~= 'table' then
        error("http endpoint 'metrics' must be a table, got " .. type(metrics), 5)
    end

    if is_array(metrics) then
        error("http endpoint 'metrics' must be a map, not an array", 5)
    end

    if metrics.enabled ~= nil and type(metrics.enabled) ~= 'boolean' then
        error("http endpoint metrics 'enabled' must be a boolean, got " .. type(metrics.enabled), 5)
    end
```

Reply:
Done.